src-tauri/src/main.rs

```diff
@@ -1214,31 +1209,6 @@ fn main() {
                 }
             }
         })
-        .on_page_load(|webview, _ | {
```

Comment:
Bummer. I thought `on_page_load` was great. How come this fails to work as expected?